packages/ui/tailwind-utils-config/components/alert.ts

```diff
@@ -0,0 +1,53 @@
+export default {
+  '.cn-alert': {
+    gap: 'var(--cn-alert-gap)',
+    paddingBlock: 'var(--cn-alert-py)',
```

Comment:
padding-block depends on the writing mode, but below you use paddings left and right - we better use the same type of properties for each padding